couchbase/admin/bucket/v1/bucket.proto

```diff
@@ -82,6 +88,7 @@ message CreateBucketRequest {
   optional couchbase.kv.v1.DurabilityLevel minimum_durability_level = 10;
   optional StorageBackend storage_backend = 11;
   optional ConflictResolutionType conflict_resolution_type = 12;
+  optional HistoryRetentionSettings history_retention_settings = 13;
```

Comment:
This should probably be individual options rather than being embedded in a message based on our existing Protostellar guidelines.